### ISSUE TITLE
Bubble columns only move you relatively up or down

### ISSUE THREAD

**Issue by Ch1216** [qouteall/GravityChanger#24]
### Minecraft version

1.20.2

### Describe the issue

Bubble columns move you up(soul sand) and down(magma block) (relatively), this works great in down gravity
however on other gravitys it does not, 
In up gravity you would expect a magma block bubble column to move you away downwards, but it moves the player drags the player upwards. same as with soul sand you would expect it to move the player away down from it, but since the player is upside down it drag the player up towards it
this also impacts other gravitys
TL:DR: Gravity should change how bubble columns move you, rather than just pushing the player relatively up or down
